com.microsoft.mrtk.core/Editor/Inspectors/PropertyDrawers/VariableRangeAttributeDrawer.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [CustomPropertyDrawer(typeof(VariableRangeAttribute))]
+    public class VariableRangePropertyDrawerer : PropertyDrawer
+    {
+        private GUIContent dropLabel;
+        private GUIStyle labelStyle;
+
+        public VariableRangePropertyDrawerer()
```

Comment:
Does this need an explicit empty constuctor?

Reply:
Mistaken copy pasta'd 🍝